Update Readme with manual check instructions (#68)
canonical/praecepta: README.md

The Vale linter operates from a series of rules. These are defined in individual YAML files, grouped into 'Styles'.
This repository contains the Canonical set of rules, or the Canonical Style. 

### Manual check

To manually check your documentation with Vale rules use the following steps:

1. Install Vale.
2. Clone this repository.
3. Run Vale with the config from this repository for testing your documentation source files: 
```shell
vale --config ~/test/praecepta/vale.ini ~/test/karapace-operator/docs/

> ```

Reviewer: This probably needs indentation (since it's part of step 3).

Author: Fixed in https://github.com/canonical/praecepta/pull/68/commits/1b3994b1e08d9aedb43a5880e7c68d8c7363b200

The Vale linter operates from a series of rules. These are defined in individual YAML files, grouped into 'Styles'.
This repository contains the Canonical set of rules, or the Canonical Style. 

### Manual check

> To manually check your documentation with Vale rules use the following steps:

Reviewer: ```suggestion
To manually check your documentation with Vale rules complete the following steps:
```

Author: I feel like `use` works better in this case.

1. Install Vale.
2. Clone this repository.

> 3. Run Vale with the config from this repository for testing your documentation source files: 

Reviewer: ```suggestion
3. Run Vale with the configuration from this repository for testing your documentation source files: 
```

Author: I feel like 
```suggestion
3. Run Vale with the configuration file `vale.ini` from this repository for testing your documentation source files: 
```
